src/Db/Adapter/SqliteAdapter.php

```diff
@@ -1098,8 +1098,12 @@ protected function calculateNewTableColumns(string $tableName, string|false $col
             }
         }
 
-        $selectColumns = array_filter($selectColumns, 'strlen');
-        $writeColumns = array_filter($writeColumns, 'strlen');
+        $selectColumns = array_filter($selectColumns, function ($value) {
+            return strlen($value) > 0;
+        });
+        $writeColumns = array_filter($writeColumns, function ($value) {
+            return strlen($value) > 0;
+        });
```

Comment:
Jezz.. why do you have to dig up old wounds :stuck_out_tongue:.

Could perhaps use this instead

```suggestion
        $selectColumns = array_filter($selectColumns, fn ($value) => $value !== '');
        $writeColumns = array_filter($writeColumns, fn ($value) => $value !== '');
```

Reply:
let see 😁